src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java

```diff
@@ -0,0 +1,93 @@
+package org.jsmart.zerocode.core.runner.parallel;
+
+import org.jsmart.zerocode.core.domain.LoadWith;
+import org.jsmart.zerocode.core.domain.TestMapping;
+import org.junit.runner.Description;
+import org.junit.runner.notification.Failure;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.ParentRunner;
+import org.junit.runners.model.InitializationError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.runner.Description.createTestDescription;
+
+public class ZeroCodeLoadRunner extends ParentRunner<TestMapping> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZeroCodeLoadRunner.class);
+    public static final String LOAD_LABEL = "<<Load>>";
+
+    private final Class<?> testClass;
+    private LoadProcessor loadProcessor;
+    private String loadPropertiesFile;
+    private Description testDescription;
+
+    public ZeroCodeLoadRunner(Class<?> testClass) throws InitializationError {
+        super(testClass);
+        this.testClass = testClass;
+        this.loadPropertiesFile = validateAndGetLoadPropertiesFile();
+        loadProcessor = new LoadProcessor(loadPropertiesFile);
+    }
+
+    @Override
+    protected List<TestMapping> getChildren() {
+        validateAnnotationPresence();
+        return Arrays.asList(testClass.getAnnotationsByType(TestMapping.class));
+    }
+
+    @Override
+    protected Description describeChild(TestMapping child) {
+        this.testDescription = createTestDescription(testClass, LOAD_LABEL + child.testMethod());
+        return testDescription;
+    }
+
+    @Override
+    protected void runChild(TestMapping child, RunNotifier notifier) {
+        notifier.fireTestStarted(testDescription);
+
+        boolean failed = loadProcessor
```

Comment:
can you rename to `didFail` or `isFailed`

Reply:
changed to `hasFailed`, sounds better.